web/admin/src/ContentEditor.js

```diff
@@ -47,7 +47,7 @@ export default class ContentEditor extends PureComponent {
             <ContentPreview content={[content]} surveys={surveys} hidden={true}/>
           </div>
           <ContentDetailsEditor content={content} onUpdate={onUpdate} surveys={surveys} />
-          <Link to="/" className="button-big">Done</Link>
+          <Link to="/" className="button-big">Save Content</Link>
```

Comment:
Note that content is updated (in unpublished state) regardless of whether this button is clicked, so this may be misleading

Reply:
I agree. I will mention that to QAS and close the ticket